sites/fast-website/src/public/index.html

```diff
@@ -3,11 +3,14 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
-    <fast-design-system-provider use-defaults>
+    <website-design-system-provider id="body-provider" class="body-provider"
```

Comment:
Just a note that if this is temporary because we are missing something from the `fast-design-system-provider` can you add a comment in here with "TODO" so we can scrape it later and clean it up.